qp_shogun/sortmerna/utils.py

```diff
@@ -27,13 +60,17 @@ def _per_sample_ainfo(
             if exists(fname):
                 if fname.endswith('R1.fastq'):
                     ftype = 'raw_forward_seqs'
+                    # zipping output files
+                    fname_compressed = _gzip_compress(fname)
                 elif fname.endswith('R2.fastq'):
                     ftype = 'raw_reverse_seqs'
+                    # zipping output files
+                    fname_compressed = _gzip_compress(fname)
                 else:
                     # this should never happen and it's not really possible
                     # to reproduce so no tests!
                     raise ValueError('File %s has an unexpected name' % fname)
-                files.append((fname, ftype))
+                files.append((fname_compressed, ftype))
```

Comment:
Since both branches imply compression, this can be rewritten like this:

```python
                  if fname.endswith('R1.fastq'):
                      ftype = 'raw_forward_seqs'
                  elif fname.endswith('R2.fastq'):
                      ftype = 'raw_reverse_seqs'
                  else:
                      # this should never happen and it's not really possible
                      # to reproduce so no tests!
                      raise ValueError('File %s has an unexpected name' % fname)

                  # zipping output files
                  fname_compressed = _gzip_compress(fname)
                  files.append((fname_compressed, ftype))
```